docs/source/error_code_list2.rst

```diff
@@ -481,3 +481,51 @@ Example:
         @override
         def g(self, y: int) -> None:
             pass
+
+
+.. _code-unimported-reveal:
+
+Check that ``reveal_type`` is imported from typing or typing_extensions [unimported-reveal]
+-------------------------------------------------------------------------------------------
+
+MyPy used to have ``reveal_type`` as a special builtin
```

Comment:
It should be `Mypy` here. The unwritten convention is (well, now was) to capitalize "mypy" as a regular word (according to its position in a sentence).